web_widget_x2many_2d_matrix/static/src/scss/web_widget_x2many_2d_matrix.scss

```diff
@@ -0,0 +1,9 @@
+.o_field_x2many_2d_matrix .row-total {
+    font-weight: bold;
+}
+  
+.o_x2many_2d_matrix {
+    thead > tr > th {
+        white-space: pre-line;
```

Comment:
You should insert a comment here to explain your use case, which is quite specific AFAICS, if you don't want it to disappear in the next migration by an innocent migrator.

Reply:
The list has its headers processed in the same way, I'm not even sure how to document it. "It should always have been like that"? :)

Reply:
Mmm maybe "Respect newline characters in headers".